start.sh

```diff
@@ -0,0 +1,14 @@
+mkdir -p outputs
+
+# get params from config.json file
+export CONFIG="$(cat config.json | jq -r .)"
```

Comment:
Worried that adding jq dependency will not allow us to merge into arch center due to not having compatibility with Windows.